<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApGKursClass</class>
 <widget class="QMainWindow" name="ApGKursClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ApGKurs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>783</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnReadFile">
           <property name="text">
            <string>Відкрити файл...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Сохранити файл</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="txtVertexTo"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="txtVertexFrom"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Кінцева вершина</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Початкова вершина</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;а) Найкоротший шлях з А в Б;&lt;br/&gt;б) Всі можливі шляхи з А в Б;&lt;br/&gt;в) Мінімальні відстані від А до будь-якого з міст;&lt;br/&gt;г) Всі міста, не поф'язані між собою дорогами;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnTaskA">
             <property name="text">
              <string>Задача А</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTaskB">
             <property name="text">
              <string>Задача Б</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTaskC">
             <property name="text">
              <string>Задача В</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTaskD">
             <property name="text">
              <string>Задача Г</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnReadFile</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>btnTaskD</tabstop>
 </tabstops>
 <resources>
  <include location="ApGKurs.qrc"/>
 </resources>
 <connections/>
</ui>
